front/src/modules/ui/object/record-table/record-table-cell/components/RecordTableCellContainer.tsx

```diff
@@ -114,6 +116,7 @@ export const TableCellContainer = ({
       <StyledCellBaseContainer
         onMouseEnter={handleContainerMouseEnter}
         onMouseLeave={handleContainerMouseLeave}
+        style={{ width }}
```

Comment:
Hey @charlesBochet,

The virtualizer's requirement for absolute positioning is affecting the table body layout. As a result, the natural dimensions are compromised, and this impacts the cell widths. To address this, I've manually adjusted the widths to match those of the table header cells.